draft-ietf-rtcweb-jsep.xml

```diff
@@ -2765,8 +2796,16 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
                   "sendonly", prepare to start transmitting media using the specified primary SSRC
                   and one of the selected payload types, once the underlying transport
                   layers have been established. Otherwise, stop transmitting RTP media,
-                  although RTCP should still be sent, as described in
-                  <xref target="RFC3264"/>, Section 5.1. </t>
+                  although RTCP should still be sent, as described in 
+                  <xref target="RFC3264"/>, Section 5.1. If RID values are specified, include the RID header
+		  extension in the RTP streams, as indicated in <xref
```

Comment:
I'd like to make the using the RTP Hdr Ext form of RID here a MUST so change to say something like "MUST include" instead of just include. (actually, I think you meant it had to do this but just want it to be very clear)